docs/how-tos/integrating-with-webflow.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 7
+---
+
+# Integrating Eppo with Webflow
+
+
+In this tutorial, you will learn how how run landing page tests in Webflow after minimal  engineering setup. At a high level, you will add a client side version of Eppo’s Javascript SDK as a script in your Webflow workspace that will:
+
+- initialize the Eppo JS client
+- handle experiment assignment logic
+- dynamically update pre-defined content variables such as text and image URLs
+
+## Prerequisites:
+
+- Eppo Account
+- Webflow account
+- Client-side analytics platform like Segment, Amplitude, Adobe, Mixpanel, etc to capture Eppo assignment events and send them to your data warehouse for analysis.
```

Comment:
Either capitalize the first bullet points, or do not capitalize "client-side analytics"